apps-rendering/src/components/headerImage.tsx

```diff
@@ -78,6 +78,20 @@ const imgStyles = (width: number, height: number): SerializedStyles => css`
 	}
 `;
 
+const liveblogImgStyles = (
+	width: number,
+	height: number,
+): SerializedStyles => css`
+	display: block;
+	width: 100%;
+	height: calc(100vw * ${height / width});
+
+	${from.desktop} {
+		width: 700px;
+		height: ${(700 * height) / width}px;
```

Comment:
the `height` & `width` are passed as parameters and their values are retrieved from `image.width` & `image.height` and image is `item.mainMedia.image`. These values are in `px` . Do you think they can be converted to rem?

Reply:
Hmm, #3547 actually duplicates this work 😄. As it happens, I think because `Img` from `image-rendering` is used under the hood here, the width and height CSS should be taken care of for you automatically. There's almost no CSS introduced in that PR to get it to work.

Given that we may well replace this with the changes from there, perhaps let's not worry about it too much for now.

Reply:
Ah yess you're right. Actually I only made this change to create a nice enough overal look since the purpose of this PR was mainly metadata 🙂  We will look at this again after #3547 is merged